.rubocop.yml

```diff
@@ -14,3 +17,15 @@ Metrics/BlockLength:
 Lint/AmbiguousBlockAssociation:
   # REVIEW: why is that dangerous?
```

Comment:
It wants you to put parens around the lambda...
Yeah, I guess we can do without this rule.